.github/workflows/test-and-lint.yml

```diff
@@ -4,9 +4,9 @@ name: Build and Test
 
 on:
   push:
-    branches: [ "main" ]
+    branches: [ "main", "v1.4.x" ]
```

Comment:
Having to update this going forward seems annoying. Maybe we could standardize on a format for version branches so we can just [use a glob](https://docs.github.com/en/actions/using-workflows/triggering-a-workflow#using-filters-to-target-specific-branches-for-pull-request-events) to target them instead of updating this every release?

Reply:
Agreed. Updated.